### PR TITLE
cmd, core, params: add support for Goerli

### PR DESCRIPTION
**Pull request by karalabe**
This PR bakes in the Görli testnet into the set officially supported ones.

### CODE REVIEW

File: cmd/utils/flags.go [hunk 1]
```diff
@@ -140,6 +140,10 @@ var (
 		Name:  "rinkeby",
 		Usage: "Rinkeby network: pre-configured proof-of-authority test network",
 	}
+	GoerliFlag = cli.BoolFlag{
+		Name:  "goerli",
+		Usage: "Goerli network: pre-configured proof-of-authority test network",
```
Comment (5chdn) [inline cmd/utils/flags.go]: nit: can you use `Görli` in strings (assuming all terminals support extended ascii in 2019)

File: params/bootnodes.go [hunk 2]
```diff
@@ -47,6 +47,26 @@ var RinkebyBootnodes = []string{
 	"enode://b6b28890b006743680c52e64e0d16db57f28124885595fa03a562be1d2bf0f3a1da297d56b13da25fb992888fd556d4c1a27b1f39d531bde7de1921c90061cc6@159.89.28.211:30303", // AKASHA
 }
 
+// GoerliBootnodes are the enode URLs of the P2P bootstrap nodes running on the
+// Goerly test network.
```
Comment (5chdn) [inline params/bootnodes.go]: nit: Görli or Goerli

File: cmd/utils/flags.go (hunk 1 above)
Comment (karalabe) [reply cmd/utils/flags.go]: Fixed!

File: params/bootnodes.go (hunk 2 above)
Comment (karalabe) [reply params/bootnodes.go]: Fixed!

**Comment by 5chdn** [comment]
Görli Genesis for the network launch tomorrow:

https://github.com/goerli/testnet/blob/master/geth/goerli.genesis

Bootnodes: 

https://github.com/goerli/testnet/blob/master/bootnodes.txt

**Comment by 5chdn** [comment]
Thanks!

**Pull request merged**